recipe-server/client/control_new/components/recipes/RecipeDetails.js

```diff
@@ -14,7 +14,7 @@ export default class RecipeDetails extends React.Component {
 
     return (
       <div className="recipe-details">
-        <Card key="recipe-details" title="Recipe">
+        <Card className="card-no-hover" key="recipe-details" title="Recipe">
```

Comment:
i would maybe just add the `noHovering` in so it just works when we upgrade Ant.